docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,136 @@
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+   >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names) page. The receiver subscribes to the event type to consume the events.
+
+   >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
+
+4. Apply your emitter Function:
+
+    ```bash
+    kyma apply function
+    ```
+   Your Function is now built and deployed in Kyma runtime. Kyma exposes it through the API Rule. The incoming payloads are processed by your emitter Function. It then sends the sanitized content to the workload that subscribes to the selected event type. In our case, it's the receiver Function.
+
+5. Test the first Function. Send the payload and see if your HTTP traffic is accepted:
+
+      ```bash
+      export KYMA_DOMAIN={KYMA_DOMAIN_VARIABLE}
+      
+      curl -X POST https://incoming.${KYMA_DOMAIN}
+      -H 'Content-Type: application/json'
+      -d '{"foo":"bar"}'
+      ```
+### Create the receiver Function
+
+1. Go to your `receiver` folder and run Kyma CLI `init` Kyma CLI command to initialize the scaffold for your second Function:
+   ```bash
+   kyma init function
+   ```
+2.  The `init` command creates the same files as in the `emitter` folder.
+3. In the `config.yaml` file, configure event types your Function will subscribe to:
+    ```yaml
+    name: event-receiver
+    namespace: default
+    runtime: nodejs14
+    source:
+       sourceType: inline
+    subscriptions:
+       - name: event-receiver
+         protocol: ""
+         filer:
+            filters:
+               - eventSource:
+                   property: source
+                   type: exact
+                   value: ""
+                eventType:
+                   property: type
+                   type: exact
+                   value: sap.kyma.custom.acme.payload.sanitised.v1
+    ```
+4.  Apply your receiver Function:
+     ```bash
+     kyma apply function
+     ```
+   The Function is configured, built, and deployed in Kyma runtime. The Subscription becomes active and all events with the selected type are processed by the Function.  
+
+5.  Test the whole setup  
```

Comment:
```suggestion
### Test the whole setup  
```

Reply:
I assume this is not part of the `Create the receiver Function` paragraph, rather a new one?